backends/tfhe-cuda-backend/cuda/include/programmable_bootstrap.h

```diff
@@ -411,6 +411,12 @@ __device__ inline int get_start_ith_ggsw(int i, uint32_t polynomial_size,
                                          int glwe_dimension,
                                          uint32_t level_count);
 
+template <typename T>
+__device__ const T *get_ith_mask_kth_block(const T *ptr, int i, int k,
+                                           int level, uint32_t polynomial_size,
+                                           int glwe_dimension,
+                                           uint32_t level_count);
+
 template <typename T>
 __device__ T *get_ith_mask_kth_block(T *ptr, int i, int k, int level,
```

Comment:
Do we need to keep this one?

Reply:
I tried but I couldn't, because we use the non-const version to calculate the output ptr of the keybundle, and therefore we also need the others you mentioned